docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,597 @@
+
+#### Enforce CHAR/VARCHAR precision when outputting to a Sink 
+
+##### [FLINK-24753](https://issues.apache.org/jira/browse/FLINK-24753)
+
+`CHAR`/`VARCHAR` lengths are enforced (trimmed/padded) by default now before entering
+the table sink.
+
+#### Support the new type inference in Scala Table API table functions 
+
+##### [FLINK-26518](https://issues.apache.org/jira/browse/FLINK-26518)
+
+Table functions that are called using Scala implicit conversions have been updated 
+to use the new type system and new type inference. Users are requested to update 
+their UDFs or use the deprecated `TableEnvironment.registerFunction` to restore 
+the old behavior temporarily by calling the function via name.
+
+#### Propagate executor config to TableConfig 
+
+##### [FLINK-26421](https://issues.apache.org/jira/browse/FLINK-26421)
+
+`flink-conf.yaml` and other configurations from outer layers (e.g. CLI) are now 
+propagated into `TableConfig`. Even though configuration set directly in `TableConfig` 
+has still precedence, this change can have side effects if table configuration 
+was accidentally set in other layers.
+
+#### Remove pre FLIP-84 methods 
+
+##### [FLINK-26090](https://issues.apache.org/jira/browse/FLINK-26090)
+
+The previously deprecated methods `TableEnvironment.execute`, `Table.insertInto`,
+`TableEnvironment.fromTableSource`, `TableEnvironment.sqlUpdate`, and 
+`TableEnvironment.explain` have been removed. Please use 
+`TableEnvironment.executeSql`, `TableEnvironment.explainSql`, 
+`TableEnvironment.createStatementSet`, as well as `Table.executeInsert`, 
+`Table.explain` and `Table.execute` and the newly introduces classes 
+`TableResult`, `ResultKind`, `StatementSet` and `ExplainDetail`.
+
+#### Fix parser generator warnings 
+
+##### [FLINK-26053](https://issues.apache.org/jira/browse/FLINK-26053)
+
+`STATEMENT` is a reserved keyword now. Use backticks to escape tables, fields and 
+other references.
+
+#### Expose uid generator for DataStream/Transformation providers 
+
+##### [FLINK-25990](https://issues.apache.org/jira/browse/FLINK-25990)
+
+`DataStreamScanProvider` and `DataStreamSinkProvider` for table connectors received 
+an additional method that might break implementations that used lambdas before. 
+We recommend static classes as a replacement and future robustness.
+
+#### Add new STATEMENT SET syntax 
+
+##### [FLINK-25392](https://issues.apache.org/jira/browse/FLINK-25392)
+
+It is recommended to update statement sets to the new SQL syntax: 
+
+```SQL
+EXECUTE STATEMENT SET BEGIN ... END;
+EXPLAIN STATEMENT SET BEGIN ... END;
+```
+
+#### Check & possible fix decimal precision and scale for all Aggregate functions 
+
+##### [FLINK-24809](https://issues.apache.org/jira/browse/FLINK-24809)
+
+This changes the result of a decimal `SUM()` with retraction and `AVG()` between
+1.14.0 and 1.14.1. It restores the behavior of 1.13 to be consistent with Hive/Spark.
```

Comment:
So...if you're upgrading from 1.13.X/1.14.1+ nothing changes?

Reply:
The issue itself is merged to 1.14 / 1.13, thus I think for high minor version of 1.13.x it should be such case. I'll remove the part irrelevant to 1.15.

Reply:
By checking with the SQL guys, the story would be like 
- If we mark 1.13 as the base
- In 1.14, we do some modification, makes it partly compatible with Hive/spark and partly compatible with sqlserver
- In 1.15, we revert the part that compatible with sqlserver back to the behavior of 1.13, makes it compatible with Hive/Spark as a whole

I'll try modifying the release notes.